tests/probes/file/CMakeLists.txt

```diff
@@ -1,3 +1,4 @@
 if(ENABLE_PROBES_UNIX)
 	add_oscap_test("test_probes_file.sh")
+	add_oscap_test("test_probes_file_multiple_file_paths.sh")
```

Comment:
As you mentioned, test will start to fail as soon as we add strace dependency. Does it make sense to enable it in CI as it will be always failing? Or do you plan to fix the probe behavior within this PR?

Reply:
I have added strace a couple minutes ago and it started to fail. I think the idea is to have a test available before we start implementing the fix. We plan to push the fix into this PR so then this test will start to pass.